client/state/selectors/test/get-account-recovery-reset-request-error.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { getAccountRecoveryResetRequestError } from '../';
+
+describe( 'getAccountRecoveryResetRequestError()', () => {
+	it( 'should return the error field in the request-reset substate treee.', () => {
```

Comment:
This might be overkill but maybe we should add a test for what it returns if there is no error